### PR TITLE
Add RuntimeConfig and DepsFile to xunit command.

### PR DESCRIPTION
**Pull request by alexperovich**
@AdamYoblick this change should fix running xunit tests on netcoreapp3.0

### CODE REVIEW

**Comment by AdamYoblick** [comment]
Thanks Alex, will give it a shot in a bit

**Comment by alexperovich** [comment]
@jonfortescue PTAL

**Comment by jonfortescue** [comment]
@alexperovich Looks great!

File: src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs [hunk 1]
```diff
@@ -101,6 +101,18 @@ private async Task<ITaskItem> PrepareWorkItem(ITaskItem xunitProject)
             string correlationPayload = IsPosixShell ? "$HELIX_CORRELATION_PAYLOAD" : "%HELIX_CORRELATION_PAYLOAD%";
             string xUnitRunner = $"{correlationPayload}/tools/{runtimeTargetFramework}/{runnerName}";
 
+            if (runtimeTargetFramework.Contains("core"))
+            {
+                var assemblyBaseName = assemblyName;
+                if (assemblyBaseName.EndsWith(".dll"))
```
Comment (jonfortescue) [inline src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs]: Are we expecting cases where this isn't the case? If so, do we want to log a message here to make sure we can binlog diagnose an issue?

File: src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs [hunk 2]
```diff
@@ -101,6 +101,18 @@ private async Task<ITaskItem> PrepareWorkItem(ITaskItem xunitProject)
...
+                {
+                    assemblyBaseName = assemblyBaseName.Substring(0, assemblyBaseName.Length - 4);
+                }
+
+                driver +=
```
Comment (jonfortescue) [inline src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs]: Add some logging for this too!

File: src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs (hunk 1 above)
Comment (alexperovich) [reply src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs]: I don't expect this to happen, I just didn't want to do weird things if that is the case. I think the logging from both the helix clients and what we already binlog should be plenty if this does happen.

Comment (jonfortescue) [reply src/Microsoft.DotNet.Helix/Sdk/CreateXUnitWorkItems.cs]: Okay sounds good!

**Pull request merged**